docs/src/components/docs/shadow-preview.svelte

```diff
@@ -1,12 +1,18 @@
 <script>
+  import { onMount } from 'svelte';
   import { Popover } from 'attractions';
   import { PopoverPositions } from 'attractions/popover';
 
   export let value;
+  let preview;
+
+  onMount(function setBoxShadow() {
+    preview.style.boxShadow = value.slice(0, -1);
+  });
 </script>
 
 <Popover position={PopoverPositions.LEFT}>
-  <div class="preview" style="box-shadow: {value};" />
+  <div class="preview" bind:this={preview}>hover me</div>
```

Comment:
Just wanted to point out that this is grammatically incorrect. You hover the mouse **over** the `div`, but you don't hover the `div` itself.
Not a big deal, though

Reply:
so this should say "hover over me"?

Reply:
If you care about grammar, yes. If you think it is not really important in this situation, then never mind.